spec/controllers/mailings_controller_spec.rb

```diff
@@ -42,15 +41,15 @@
     end
 
     context 'with user logged in' do
-      let(:user) { FactoryGirl.create(:student) }
```

Comment:
is there a technical reason for that? does FactoryGirl not support the syntax anymore? the reason why I ask is that I really like the explicit method call. it obvious the everybody that this is `create()` from FactoryGirl. you don't need knowledge about the dependencies that are used in the app.